src/api/rootResolver.js

```diff
@@ -120,6 +139,34 @@ const resolvers = {
   },
 
   Mutation: {
+    async create(_, { name, deposit, limitOfParticipants}) {
+      const ethers = getEthers()
```

Comment:
This code is confusing and has redundant parts to it. Please remove all debugging-related code and simplify it.

Reply:
Are you talking about creation events?  we should either create `events` query endpoint or need other ways as currently I cannot find out what are the newly created contract address

Reply:
Ok, done and removed